src/index.css

```diff
@@ -7,6 +7,33 @@
 }
 
 
+.jp-FileBrowser-breadcrumbs {
+  padding: 5px;
```

Comment:
Yeah, I arbitrarily drew the line at all layout in src and all color/font/border stuff in example.